test/FunctionMetadataGeneratorTests/FunctionMetadataGeneratorTests.cs

```diff
@@ -303,6 +305,142 @@ void ValidateBlobOutput(ExpandoObject b)
                     { "Properties", new Dictionary<String, Object>() }
                 });
             }
+
+
+            var blobClientToBlobStringFunction = functions.Single(p => p.Name == "BlobClientToBlobStringFunction");
+
+            ValidateFunction(blobClientToBlobStringFunction, "BlobClientToBlobStringFunction", GetEntryPoint(nameof(SDKTypeBindings), nameof(SDKTypeBindings.BlobClientToBlobStringFunction)),
+                b => ValidateBlobTriggerForBlobClient(b),
+                b => ValidateBlobInput(b),
+                b => ValidateBlobOutput(b));
+
+
+            void ValidateBlobTriggerForBlobClient(ExpandoObject b)
+            {
+                AssertExpandoObject(b, new Dictionary<string, object>
+                {
+                    { "Name", "blob" },
```

Comment:
This single test function is validating like 20 different scenarios across all the storage binding types. Is there a reason why these aren't broken apart into smaller test cases?

Reply:
refactored it